### PR TITLE
chore: optimize the dashboard templates

### PR DESCRIPTION
**Pull request by huimiu**
Modify the way styles are defined in the template from inline-style to CSS.

E2E TEST: N/A

### CODE REVIEW

File: templates/tab/js/dashboard/README.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 
 ## Introduction
 
-This is a Teams tab dashboard app that uses the [Fluent UI](https://react.fluentui.dev/?path=/docs/concepts-introduction--page) and the [Microsoft Graph API](https://learn.microsoft.com/en-us/graph/use-the-api) to display a user's profile information and recent Teams activity.
+This is a dashboard tab app that embed a canvas containing multiple cards that provide an overview of data or content in Microsoft Teams.
 
 ![Default theme](./public/dashboard.png)
```
Comment (hund030) [inline templates/tab/js/dashboard/README.md:7]: Could we reduce the picture size?

Comment (huimiu) [reply templates/tab/js/dashboard/README.md:7]: Sure, compressed the image.

File: templates/tab/js/dashboard/src/App.jsx [hunk 2]
```diff
@@ -1,60 +1,52 @@
...
-        }}
       >
         <Router>
           <Route exact path="/">
             <Redirect to="/tab" />
           </Route>
-          {loading ? (
-            <Spinner style={{ margin: 100 }} />
-          ) : (
+          {
```
Comment (hund030) [inline templates/tab/js/dashboard/src/App.jsx:42]: Why do we remove the loading indicator?

Comment (huimiu) [reply templates/tab/js/dashboard/src/App.jsx]: This was removed by mistake, it has been re-added, thanks for helping to find this problem.

**Review by hund030 — DISMISSED** [review]

**Review by hund030 — DISMISSED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**